iaac/azure/arm_templates/mysql_flexible_server/parameters.json

```diff
@@ -0,0 +1,88 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "serverName": {
+            "value": "infra"
+        },
+        "location": {
+            "value": "westeurope"
+        },
+        "serverEdition": {
+            "value": "Burstable"
+        },
+        "vCores": {
+            "value": 1
+        },
+        "storageSizeGB": {
+            "value": 20
+        },
+        "haEnabled": {
+            "value": "Disabled"
+        },
+        "firewallRules": {
+            "value": {
+                "rules": [
+                    {
+                        "name": "RULE_NAME",
+                        "startIPAddress": "A.B.C.D",
+                        "endIPAddress": "X.Y.Z.W"
```

Comment:
We should somehow parametrize this, e.g. with bash scripts and/or sed command

Reply:
This will depend on DNAT in AKS for our pods outgoing requests, might be tricky. But i guess we could get the parameters from kubectl - Azure CLI after AKS  creation automatically sets context to lates k8s cluster. Might work